brain/converter/crypto.py

```diff
@@ -32,6 +32,35 @@ def getsalt(app=True, root='/vagrant'):
                 return base64.b64encode(os.urandom(32))
 
 
+def getscryptparams(app=True, root='/vagrant'):
+    '''@getscryptparams
+
+    This method returns the parameters N,r,p for the scrypt function.
+    
+    Note: No minimum is enforced for these parameters because a minimum can
+    potentially break the hashing function in a system incapable of meeting
+    those requirements
+
+    @app, indicates if function is to be used by application, or manually
+
+    '''
+    if app:
+        N = current_app.config.get('SCRYPT_N')
+        r = current_app.config.get('SCRYPT_R')
+        p = current_app.config.get('SCRYPT_P')
+	return pow(2,N), r, p
+    else:
+        with open(root + "/hiera/settings.yaml", 'r') as stream:
+            try:
+           	yamlstream = yaml.load(stream)
+                N = yamlstream['crypto']['scrypt_n']
+                r = yamlstream['crypto']['scrypt_r']
+                p = yamlstream['crypto']['scrypt_p']
+                return pow(2,N), r, p

```

Comment:
This syntax is more dense then my typical `dict` structure I use, for returning multiple values (but I like it).  I would include in the docblock, in the `getscryptparams` function, what each parameter represents, including `pow(2,N)`.  Additionally, the PEP8 linter may complain, and require a space between the `2`, and `N` in `pow(2,N)`.